packages/dicomImageLoader/src/imageLoader/convertColorSpace.ts

```diff
@@ -23,6 +23,25 @@ function convertYBRFull(imageFrame, colorBuffer, useRGBA) {
   }
 }
 
+/**
+ * This function checks color space conversion data requirements before
+ * apply them
+ * @param imageFrame
+ * @param RGBA
+ * @returns
+ */
+export function isColorConversionRequirementsFulfilled(imageFrame, RGBA) {
+  const { rows, columns } = imageFrame;
+  if (
+    imageFrame.photometricInterpretation === 'YBR_FULL_422' &&
+    imageFrame.pixelDataLength === 2 * rows * columns
```

Comment:
Specifically, the rows element must be even length, and the total length be 2 * rows * columns